### ISSUE TITLE
Decreased serum iduronate sulfatase level

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7432]
**Preferred term label:**
Decreased serum iduronate sulfatase level

**Synonyms**

**Definition (free text, please give PubMed ID)**
An idecreased concentration of iduronate-2-sulfatase activity in the blood circulation.

**Parent term (use hpo.jax.org/app)**
Sibling of [Increased serum iduronate sulfatase level HP:0003538](https://hpo.jax.org/app/browse/term/HP:0003538)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:309900 

1/1  PMID:35282545

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
